src/main/java/io/codearte/jfairy/producer/company/CompanyProvider.java

```diff
@@ -67,8 +68,27 @@ private void generateDomain() {
 		if (domain != null) {
 			return;
 		}
-		domain = TextUtils.stripAccents(StringUtils.strip(StringUtils.deleteWhitespace(name.toLowerCase()), ".").replace("/", ""))
-				+ "." + dataMaster.getRandomValue(DOMAIN);
+//		domain = TextUtils.stripAccents(StringUtils.strip(StringUtils.deleteWhitespace(name.toLowerCase()), ".").replace("/", ""))
```

Comment:
Yep, this feature is a little bit like Scala trait. It's convenient for code design. But I wonder if there may be a compatible problem in different Java versions. There're some features in Java8 that cannot be supported in JRE-7, and I'm not sure if this one is included.